api/v1beta2/cloudstackfailuredomain_conversion.go

```diff
@@ -30,3 +31,7 @@ func (dst *CloudStackFailureDomain) ConvertFrom(srcRaw conversion.Hub) error { /
 	src := srcRaw.(*v1beta3.CloudStackFailureDomain)
 	return Convert_v1beta3_CloudStackFailureDomain_To_v1beta2_CloudStackFailureDomain(src, dst, nil)
 }
+
+func Convert_v1beta3_CloudStackFailureDomainSpec_To_v1beta2_CloudStackFailureDomainSpec(in *v1beta3.CloudStackFailureDomainSpec, out *CloudStackFailureDomainSpec, s machineryconversion.Scope) error { // nolint
```

Comment:
curious why is this changing?
It seems like the changes don't affect the API and it's all internal

Reply:
This was a fix in upstream. I added this change in this PR while building.
This is already merged upstream. Let me rebase the PR once. https://github.com/kubernetes-sigs/cluster-api-provider-cloudstack/blob/d1a6ad96166164e991de42e9fb59da4f816ebd54/api/v1beta2/cloudstackfailuredomain_conversion.go#L35-L37